src/cobra/util/context.py

```diff
@@ -1,46 +1,68 @@
 """Context manager for the package."""
 
 from functools import partial
+from typing import Any, Callable
 
 
 class HistoryManager:
-    """Record a list of actions to be taken at a later time. Used to
-    implement context managers that allow temporary changes to a
-    :class:`~cobra.core.Model`.
+    """
+    Define a base context manager.
+
+    It records a list of actions to be taken at a later time.
+    This is used to implement context managers that allow temporary
+    changes to a `cobra.core.Model`.
 
     """
 
     def __init__(self):
-
-        # self._history just acts as a stack
+        """Initialize the class."""
```

Comment:
In principle, `__init__` should always accept `**kwargs` and call `super().__init__(**kwargs)` at least. That is to enable multiple inheritance.

Reply:
Yes. Totally missed it.